src/utils/get-options/get-assertion-timeout.ts

```diff
@@ -0,0 +1,11 @@
+import AssertionCommand from '../../test-run/commands/assertion';
+import { Dictionary } from '../../configuration/interfaces';
+
+export default function getAssertionTimeout (command: AssertionCommand, options: Dictionary<OptionValue>): number {
+    // @ts-ignore
+    const { timeout: commandTimeout } = command.options;
```

Comment:
```suggestion
    const commandTimeout = command.options?.timeout;
```

Reply:
Unfortunately, it doesn't allow us to remove `@ts-ignore` comment.
The command properties added [dynamically](https://github.com/DevExpress/testcafe/blob/master/src/test-run/commands/assertion.js#L45).

We can fix it using separate type definition, but it's better to rewrite in more object manner. Command rewriting is a big separate task and it's out of context this PR.